Rubberduck.Inspections/Concrete/UnreachableCaseInspection/ExpressionFilter.cs

```diff
@@ -0,0 +1,653 @@
+﻿using Rubberduck.Parsing.Grammar;
```

Comment:
I believe all files in `Rubberduck.Inspections/Concrete` are near-duplicates of same files in `Rubberduck.StaticCodeAnalysis/Inspections/Concrete`. There are also some differences. Please fix.

Reply:
Oh wow...my bad...really bad....thank you for spotting this.  When I did the merge a while back to the new hierarchy, the initial new files were left behind in the old folder structure...and I missed it - obviously.  Amazing what you no longer see after staring at something for months.